test/integration/remote/auth_test.py

```diff
@@ -46,25 +36,25 @@ def test_retries(self):
         self.conan = TestClient(servers=self.servers, inputs=["bad", "1",
                                                               "bad2", "2",
                                                               "nacho@gmail.com", "nachopass"])
-        save(os.path.join(self.conan.current_folder, CONANFILE), conan_content)
+        self.conan.save({"conanfile.py": GenConanfile("openssl", "2.0.1")})
         self.conan.run("export . --user=lasote --channel=testing")
-        errors = self.conan.run("upload %s -r default --only-recipe" % str(self.ref))
+        errors = self.conan.run("upload %s -r default --only-recipe -v=quiet" % str(self.ref))
```

Comment:
This ``-v=quiet`` is what made the test fail, now it is passing.

All the other changes in this file are just migration unittest=>pytest